Makefile

```diff
@@ -91,13 +91,6 @@ testshell: .env .docker-build  ## | Open shell in test environment.
 docs: .env .docker-build  ## | Build docs.
 	docker-compose run --rm --user ${USE_UID} linting docs
 
-frontend/build/:
-	make build-frontend
-
-.PHONY: build-frontend
-build-frontend:  ## | Build frontend static files.
```

Comment:
Year-ago-me added this to patch over difficulties running the tests after building Tecken. However, it doesn't build the frontend static files at all--it's a complete lie. Year-ago-me wasted today-me's time!

The problem is that the tests for some reason need stuff in `/app/frontend/build/` to exist. That exists in the `tecken:build` Docker image, but when we use `docker-compose` to run anything, we volume mount the repository directory as `/app` and thus the `/app/frontend/build/` that's in the image disappears in a puff of magic smoke.

So year-ago-me thought, "Well that sucks. I'll just create an empty directory there."

Today-me went through and did a different way to fix this and put that "if the directory don't exist, create it" bit in `bin/test.sh` and removed this completely misleading "Build frontend static files" make rule of insincerity.